src/constructs/cloudwatch/alarm.ts

```diff
@@ -9,6 +9,15 @@ export interface GuAlarmProps extends AlarmProps, AppIdentity {
   snsTopicName: string;
 }
 
+export interface Http5xxAlarmProps
```

Comment:
Users now use this same interface to configure a 5XX alarm, regardless of whether they are using EC2 or Lambda for compute.